app/assets/stylesheets/application.css

```diff
@@ -1,34 +1,35 @@
 /*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
- * or vendor/assets/stylesheets of plugins, if any, can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the top of the
- * compiled file, but it's generally better to create a new file per style scope.
- *
- * If there are other assets (which are not in this file) that are being precompiled by
- * 'stylesheet_link_tag' command in views, then add them on /config/initializers/assets.rb
+* This is a manifest file that'll be compiled into application.css, which will include all the files
```

Comment:
I'm a little worried about the spacing changes on this page, as I believe Sprockets requires very specific spacing. Just not sure! Would you mind restoring the original spacing just in case? Thanks!

Reply:
The spacing is correct here... Yes I got a Sprockets error.. Then I changed the spacing back to normal.. Now through the commit, its the same as normal.